packages/fe-container/src/__tests__/App.test.tsx

```diff
@@ -105,20 +105,17 @@ describe('App test', () => {
   it('toggle is working', async () => {
     const toggle = await screen.findByTestId('top-bar-side-menu-toggle')
 
-    // @ts-ignore
-    const retrieveEntryClasses = () => screen.getByText('Href entry').parentElement.parentElement.parentElement.parentElement.classList
-
     expect(global.window.document.title).toEqual('My Company')
 
-    expect(retrieveEntryClasses()).not.toContain('opened')
+    expect(document.getElementsByClassName('ant-drawer-open').length).toBe(0)
```

Comment:
can you keep the method invocation? so we don't have to repeat this on every line